lib/main.js

```diff
@@ -235,10 +261,19 @@ export default {
       }
     }
 
-    const linter = new Linter(Object.assign({
-      formatter: 'json',
-      rulesDirectory,
-    }, options));
+    const program = await this.getProgram(Linter, configurationPath);
+    let linter;
+    if (program) {
```

Comment:
Nope! Thanks. 😊 Removed in https://github.com/AtomLinter/linter-tslint/pull/157/commits/60be059ed7ad45ed87eb78ed6ac1e0eb94c252e9